core/domain/collection_services.py

```diff
@@ -712,19 +698,16 @@ def _save_collection(committer_id, collection, commit_message, change_list):
 
     collection_model = collection_models.CollectionModel.get(
         collection.id, strict=False)
-    if collection_model is None:
```

Comment:
Collection model cannot be none as we are passing the collection as a parameter. Also this function is called by update_collection which only works if the collection is put into the datastore

Reply:
Done